std/fs/copy.ts

```diff
@@ -22,18 +22,18 @@ async function ensureValidCopy(
   dest: string,
   options: CopyOptions,
   isCopyFolder = false
-): Promise<Deno.FileInfo> {
-  let destStat: Deno.FileInfo | null;
+): Promise<Deno.FileInfo | null> {
+  let destStat: Deno.FileInfo | null = null;
 
   try {
     destStat = await Deno.lstat(dest);
   } catch (err) {
     if (err instanceof Deno.DenoError && err.kind == Deno.ErrorKind.NotFound) {
-      return;
+      return destStat;
     }
   }
 
-  if (isCopyFolder && !destStat.isDirectory()) {
+  if (isCopyFolder && !destStat!.isDirectory()) {
```

Comment:
`assert(destStat)` prior to this...

Reply:
Is it ok to use the `assert` from `std/testing/asserts.ts` here or should that function be copied into a `util` file like the one in `cli/js`?